component-params-env.yaml

```
@@ -24,12 +24,12 @@ known_issues:
 - image: RELATED_IMAGE_DSP_TOOLBOX_IMAGE
   jira: https://redhat.atlassian.net/browse/RHOAIENG-60396
   reason: Removed with InstrutLab pipeline removal since RHOAI 3.0
-- image: RELATED_IMAGE_CLI_IMAGE
-  jira: https://redhat.atlassian.net/browse/RHOAIENG-60519
-  reason: image is not used in monitoring templates and should be removed
 - image: RELATED_IMAGE_ODH_CAIKIT_NLP_IMAGE
   jira: https://redhat.atlassian.net/browse/RHOAIENG-60523
   reason: caikit-nlp support is removed since RHOAI 3.0
+  image: RELATED_IMAGE_DSP_TOOLBOX_IMAGE
```

YAMLlint (1.37.1)

[error] duplication of key "image" in mapping

(key-duplicates)